src/main/java/org/zephyrsoft/sdb2/model/SongsModel.java

```diff
@@ -48,15 +49,14 @@
 	
 	private static final Logger LOG = LoggerFactory.getLogger(SongsModel.class);
 	
-	@XmlElement(name = "autoSort")
-	private boolean autoSort = true;
+	private boolean autoSort = false;
 	
 	@XmlElement(name = "song")
 	private List<Song> songs = null;
 	
 	private List<TransparentListModel<Song>> createdListModels = null;
 	
-	private List<SongsModelListener> songsModelListeners = null;
+	private ArrayList<SongsModelListener> songsModelListeners = null;
```

Comment:
Please only use the interface (`List`) as variable type. The actual implementation should be unimportant.

Reply:
Of course. Changed it.